android/src/main/java/com/thebluealliance/androidclient/binders/TeamInfoBinder.java

```diff
@@ -118,6 +125,13 @@ public void updateData(@Nullable TeamInfoBinder.Model data) {
             string.setSpan(new TextAppearanceSpan(mActivity,
                     R.style.InfoItemLabelStyle), 0, 3, Spannable.SPAN_INCLUSIVE_INCLUSIVE);
             teamFullName.setText(string);
+            teamFullName.setMaxLines(TEAM_FULL_NAME_COLLAPSED_MAX_LINES);
+            teamFullName.setEllipsize(TextUtils.TruncateAt.END);
```

Comment:
... and at least one known Android bug: [Issue 2254](https://code.google.com/p/android/issues/detail?id=2254).